src/main/java/com/synopsys/integration/alert/channel/ChannelTemplateManager.java

```diff
@@ -43,7 +42,6 @@
 import com.synopsys.integration.alert.database.audit.AuditNotificationRepository;
 import com.synopsys.integration.alert.database.audit.relation.AuditNotificationRelation;
 
-@Transactional
```

Comment:
Removing the annotation from the class level should increase efficiency of methods that do not interact with the DB.